### ISSUE TITLE
Does it works 24hours time format?

### ISSUE THREAD

**Issue by armandchard** [sidaudhi/angular-circular-timepicker#1]
Hi, 
I like your work but I need to use the time format 24 hours. 
Is it implemented or not? 
If yes, how I can use it?

Regards.

**Comment by sidaudhi** [comment]
Will have it implemented soon.

**Comment by armandchard** [comment]
Ok nice, I'll watch the project and wait this upfrade. Thank you !

**Comment by sidaudhi** [comment]
This is now available through the format parameter (which expects a JSON object):
You can use military:true for 24 hour format:
`<datetimepicker model="datetime2" format="{military:true}"></datetimepicker>`

You can alternatively use momentFormat to directly specify the display format using momentFormat:
`<datetimepicker model="datetime2" format="{momentFormat:'YYYY'}"></datetimepicker>`
For the display format specs, refer to moment.js documentation: 
http://momentjs.com/docs/#/query/

**Comment by armandchard** [comment]
Thx a lot.
I'll try it soon.

**Comment by sidaudhi** [comment]
Great! Do close this issue once you're able to use this feature.

**Comment by armandchard** [comment]
It works fine to show the time like I would.
But I thought we can select 24hours in the timepicker but we still need to select AM or PM.
It's not a problem for me but French users of my web app it's not ok :(

**Comment by sidaudhi** [comment]
Right. I'll have to rethink how to accommodate 24 circles in the space of only 12. Maybe another round of concentric circles. I'll think of something for sure.

**Comment by armandchard** [comment]
Maybe you can switch hours and minutes. Then add a bubble (hour) between with less space than today. How do you think?
